Controllers/user.js

```diff
@@ -211,7 +189,7 @@ async function addResponders(req, res) {
 
         for (var i = 0, len = respondersToAdd.length; i < len; i++) {
           user.responders.push(respondersToAdd[i]);
-          user.save();
+          await user.save();
```

Comment:
as a note, I feel like this _might_ solve (or have something to do with) https://github.com/CodetheChangeFoundation/UHN-Backend-Server/issues/50

Reply:
@matthewleemle